src/Files.App/App.xaml.cs

```diff
@@ -226,6 +226,7 @@ private static async Task InitializeAppComponentsAsync()
 			var updateService = Ioc.Default.GetRequiredService<IUpdateService>();
 			await updateService.CheckForUpdates();
 			await updateService.DownloadMandatoryUpdates();
+			await updateService.CheckAndUpdateFilesLauncherAsync();
```

Comment:
Does this run after startup is complete?

Reply:
Called asynchronously during startup.

Reply:
Can we delay it until after startup? It probably doesn't take long but we might as well delay it.